lib/src/sdk_definitions.dart

```diff
@@ -591,11 +591,56 @@ class PrefetchOptions {
   });
 }
 
+/// [code] **LOCATION_PERMISSION_DENIED**
+/// type: [ErrorType.critical].
+///
+/// **CAUSE**: Location permissions were not granted yet,
+/// so SDK won't be able to start positioning. The permissions needed to fix this error are:
+///   * ACCESS_FINE_LOCATION (Android)
+///   * NSLocationWhenInUseUsageDescription (iOS) (at least)
+///
+/// [code] **BLUETOOTH_PERMISSION_DENIED**
+/// (Android only)
+/// type: [ErrorType.critical].
+///
+/// **CAUSE**: BLUETOOTH_CONNECT or BLUETOOTH_SCAN are not granted yet,
+/// so SDK won't be able to start positioning.
+///
+/// [code] **BLUETOOTH_SENSOR_DISABLED**
+/// type: [ErrorType.critical] for iOS but [ErrorType.nonCritical] for Android.
+///
+/// **CAUSE**: The bluetooth sensor of the device is off,
+/// so iOS will stop positioning and Android won't give a precise location as with this sensor on.
+///
+/// [code] **LOCATION_SENSOR_DISABLED**
```

Comment:
Eu cargaríame o de "SENSOR" por todas partes. Eu entendo que o servicio de ubicación é máis que un sensor (non se pode escanear wifi nin ble, etc.). Cando desactivas a ubicación estás a "negar" un montón de cousas, non sólo o GPS do dispositivo.